sky/cli.py

```diff
@@ -5097,7 +5097,8 @@ def _deploy_local_cluster(gpus: bool):
 
     # Get directory of script and run it from there
     cwd = os.path.dirname(os.path.abspath(up_script_path))
-    run_command = up_script_path + ' --gpus' if gpus else up_script_path
+    run_command = up_script_path + f' --user {common_utils.get_user_hash()}'
```

Comment:
This created ` /tmp/skypilot-kind---user.yaml`. Maybe we want to remove the `--user` here?

```
No kind clusters found.
Generating /tmp/skypilot-kind---user.yaml
D 10-07 22:21:55 skypilot_config.py:228] Using config path: /Users/romilb/.sky/config.yaml
D 10-07 22:21:55 skypilot_config.py:233] Config loaded:
D 10-07 22:21:55 skypilot_config.py:233] {'allowed_clouds': ['kubernetes', 'gcp', 'lambda', 'aws'],
D 10-07 22:21:55 skypilot_config.py:233]  'jobs': {'controller': {'resources': {'cloud': 'kubernetes',
D 10-07 22:21:55 skypilot_config.py:233]                                        'cpus': '4+',
D 10-07 22:21:55 skypilot_config.py:233]                                        'memory': '4+'}}},
D 10-07 22:21:55 skypilot_config.py:233]  'kubernetes': {'provision_timeout': -1},
D 10-07 22:21:55 skypilot_config.py:233]  'serve': {'controller': {'resources': {'cloud': 'kubernetes',
D 10-07 22:21:55 skypilot_config.py:233]                                         'cpus': '4+',
D 10-07 22:21:55 skypilot_config.py:233]                                         'memory': '4+'}}}}
D 10-07 22:21:55 skypilot_config.py:245] Config syntax check passed.
Creating cluster "skypilot" ...
 • Ensuring node image (kindest/node:v1.25.3) 🖼  ...
 ✓ Ensuring node image (kindest/node:v1.25.3) 🖼
 • Preparing nodes 📦   ...
 ✓ Preparing nodes 📦 
 • Writing configuration 📜  ...
 ✓ Writing configuration 📜
 • Starting control-plane 🕹️  ...
 ✓ Starting control-plane 🕹️
 • Installing CNI 🔌  ...
 ✓ Installing CNI 🔌
 • Installing StorageClass 💾  ...
 ✓ Installing StorageClass 💾
Set kubectl context to "kind-skypilot"
You can now use your cluster with:

kubectl cluster-info --context kind-skypilot
```

Reply:
Thanks for pointing it out! I've done some command argument reordering but forgot to test it after. Updated and tested now!

```bash
No kind clusters found.
Generating /tmp/skypilot-kind-4a0782e9.yaml
Creating cluster "skypilot" ...
 • Ensuring node image (kindest/node:v1.27.3) 🖼  ...
 ✓ Ensuring node image (kindest/node:v1.27.3) 🖼
 • Preparing nodes 📦   ...
 ✓ Preparing nodes 📦 
 • Writing configuration 📜  ...
 ✓ Writing configuration 📜
 • Starting control-plane 🕹️  ...
 ✓ Starting control-plane 🕹️
 • Installing CNI 🔌  ...
 ✓ Installing CNI 🔌
 • Installing StorageClass 💾  ...
 ✓ Installing StorageClass 💾
Set kubectl context to "kind-skypilot"
```